controllers/updates/views/post/blogpost.njk

```diff
@@ -3,71 +3,73 @@
 {% from "components/inline-links/macro.njk" import inlineLinks %}
 {% from "components/hero.njk" import hero with context %}
 {% from "components/promo-card/macro.njk" import promoCard %}
+{% from "components/card/macro.njk" import card %}
 {% from "components/split-nav/macro.njk" import splitNav %}
 {% from "components/card/macro.njk" import card %}
 
 {% extends "layouts/main.njk" %}
 
 {% macro articleMeta(entry) %}
 
-        {% set aboutTitle = "Authors" if entry.authors.length > 1  else "Author" %}
-        {% call card(aboutTitle, pageAccent) %}
-            {% for author in entry.authors -%}
-                <aside class="o-media">
-                    {% if author.photo %}
-                        <img src="{{ author.photo }}"
-                             alt="{{ author.title }}"
-                             class="o-media-figure"
-                             width="60"/>
+    {% set aboutTitle = "Authors" if entry.authors.length > 1  else "Author" %}
+    {% call card(aboutTitle, pageAccent) %}
+        {% for author in entry.authors -%}
+            <aside class="o-media">
+                {% if author.photo %}
+                    <img src="{{ author.photo }}"
+                         alt="{{ author.title }}"
+                         class="o-media-figure"
+                         width="60"/>
+                {% endif %}
+                <div class="o-media-body">
+                    <p class="u-no-margin"><strong>{{ author.title }}</strong></p>
+                    {% if author.authorTitle %}
+                        <p>
+                            {{ author.authorTitle }}
+                            {% if author.shortBiography %}
+                                <br>
+                                {{ author.shortBiography}}
+                            {% endif %}
+                        </p>
                     {% endif %}
-                    <div class="o-media-body">
-                        <p class="u-no-margin"><strong>{{ author.title }}</strong></p>
-                        {% if author.authorTitle %}
-                            <p>
-                                {{ author.authorTitle }}
-                                {% if author.shortBiography %}
-                                    <br>
-                                    {{ author.shortBiography}}
-                                {% endif %}
-                            </p>
-                        {% endif %}
-                    </div>
-                </aside>
-                <p class="u-margin-top-s"><a href="{{ localify('/news/blog?author=' + author.slug) }}">See all posts by this author</a></p>
+                </div>
+            </aside>
+            <p class="u-margin-top-s"><a href="{{ localify('/news/blog?author=' + author.slug) }}">See all posts by this author</a></p>
+        {% endfor %}
+    {% endcall %}
+
+    {% call card("About this blog post", pageAccent) %}
+
+        <p class="article-meta__pubdate u-margin-bottom-s">
```

Comment:
Can this `article-meta__pubdate` class go too? It's a bit of a loose class and afaict only adds margin which the `u-` class overrides.

Reply:
Done!